src/main/resources/assets/lib/md5.js

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
rather than this, could we use SubtleCrypto.digest it doesn't provide md5 but gives sha-* which I guess is good enough for us https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest and we probably don't need to support old browsers

Reply:
I think the longer term plan was to just pre-process the axe data and inject a UUID for each violation (whilst also reducing its size by discarding fields we don't use)